src/common/sys/darwin.ts

```diff
@@ -18,7 +18,7 @@ export function resolveArduinoPath(): string {
     return result || "";
 }
 
-export function validateArduinoPath(arduinoPath: string): boolean {
+export function validateArduinoPath(arduinoPath: string, isArduinoCli = false): boolean {
     return fileExistsSync(path.join(resolveMacArduinoAppPath(arduinoPath), "/Contents/MacOS/Arduino"));

```

Comment:
```suggestion
    return fileExistsSync(path.join(resolveMacArduinoAppPath(arduinoPath), isArduinoCli ? "arduino-cli" : "/Contents/MacOS/Arduino"));

```

Reply:
Looks promising, I'll test it 👍

Reply:
I appreciate it, man! :fist:

Reply:
You will have to change your arduino.path to where the arduino-cli is installed tho